README.md

```diff
@@ -63,7 +76,7 @@ On release, automated continuous integration tests run the pipeline on a full-si
 
 ## Documentation
 
-The nf-core/blobtoolkit pipeline comes with documentation about the pipeline [usage](https://nf-co.re/blobtoolkit/usage), [parameters](https://nf-co.re/blobtoolkit/parameters) and [output](https://nf-co.re/blobtoolkit/output).
+The nf-core/blobtoolkit pipeline comes with documentation about the pipeline [usage](https://github.com/sanger-tol/blobtoolkit/blob/main/docs/usage.md), [parameters](https://github.com/sanger-tol/blobtoolkit/blob/main/docs/parameters.md) and [output](https://github.com/sanger-tol/blobtoolkit/blob/main/docs/output.md).
 
 ## Credits
```

Comment:
Need to update credits.
1. Change from `nf-core/blobtoolkit` to `sanger-tol/blobtoolkit`
2. Add your name to the list as well.
3. Give credit to Richard Challis for design/original pipeline and Sujai Kumar for maintaining + Link to the original blobtoolkit
4. Add Matthieu and me
5. Also add a sentence on how Alexander was supported by GSoC
6. Remove TODO